host/sgx/sgxquote.c

```diff
@@ -553,6 +626,10 @@ static void _load_sgx_dcap_qvl_impl(void)
 
     if (_qvl_module)
     {
+        OE_CHECK(_lookup_function(
+            _qvl_module,
+            "sgx_qv_set_enclave_load_policy",
+            (void**)&_sgx_qv_set_enclave_load_policy));
```

Comment:
why does this have an _sgx_ when the next looked up funciton pointer just has "sgx_"

Reply:
To add support for both SGX (here) and TDX(next lookup function)